lib/helpers/emitter.js

```diff
@@ -51,4 +57,35 @@ exports.Emitter = class Emitter extends EventEmitter {
       .on('snapshot', onSnapshot)
       .on('end', onEnd);
```

Comment:
Would the new valid events also require emitter functions to print them? Or is there a catch-all

Reply:
my bad, missed this